hkmc2/shared/src/test/mlscript/backlog/UCS.mls

```diff
@@ -81,19 +81,6 @@ if
 let audits = new Set()
 //│ audits = Set{}
 
-:fixme
-if audits.has(1) === true do
-  print("ok")
-else
-  1
```

Comment:
You removed the test case instead of moving it to a different file?!

Reply:
😢 Oh! I shouldn't have done this. Thanks for the catch.